Compile fix for android on Mac

Tried to compile the app on Mac, for android.
I run on the following error:
AudioOutputOpenSL.cpp:431:18: error: use of undeclared identifier 'log10' 
and
AudioOutputOpenSL.cpp:450:12: error: use of undeclared identifier 'pow' 

I solved this by adding "#include <cmath>" on AudioOutputOpenSL.cpp line 30 (inside the #ifdef Q_OS_ANDROID )

--- a/src/output/audio/AudioOutputOpenSL.cpp
+++ b/src/output/audio/AudioOutputOpenSL.cpp
@@ -27,6 +27,7 @@
 #include <SLES/OpenSLES_Android.h>
 #include <SLES/OpenSLES_AndroidConfiguration.h>
 #include <sys/system_properties.h>
+#include <cmath>
 #endif
 #include "QtAV/private/mkid.h"
 #include "QtAV/private/factory.h"

